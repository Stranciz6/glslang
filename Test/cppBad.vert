#define m#0#
#if m
