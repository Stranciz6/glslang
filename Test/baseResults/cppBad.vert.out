cppBad.vert
ERROR: 0:2: 'preprocessor evaluation' : bad expression 
ERROR: 0:2: '#if' : unexpected tokens following directive 
ERROR: 0:3: '' : missing #endif 
ERROR: 3 compilation errors.  No code generated.


Shader version: 100
ERROR: node is still EOpNull!
0:?   Linker Objects


Linked vertex stage:

ERROR: Linking vertex stage: Missing entry point: Each stage requires one "void main()" entry point

Shader version: 100
ERROR: node is still EOpNull!
0:?   Linker Objects

